### ISSUE TITLE
Client performance drop on failed to load.

### ISSUE THREAD

**Issue by LuaTenshi** [Facepunch/garrysmod-requests#785]
Seems that when a file/sound/texture is consistently not found the game keeps attempting to grab it on every instance. This is especially true with sounds. 

To quote [Bletotum](https://github.com/Facepunch/garrysmod-requests/issues/783#issuecomment-233564349)

> The issue isn't that anyone wants error messages removed, but that the event causing the error (the missing file trying to load) is supposedly the cause of the lag and should not occur multiple times.
> 
> If this is false and the console error message is the culprit of the lag, then the message should only display once per missing file per session.

I will experiment further trying to zero in on the true cause of the performance drop. But I do feel that a discussion on this is better started sooner then later. Because this issue has been in Garry's mod for years.

---

**TL;DR Client seems to get lag when constantly getting the "failed to load" error message.**
**So I am suggesting having the sound attempt to load only once instead of over and over again, because I assume that may be the cause of the lag.**

---
## Control Image. (Regular server with not much going on.)

![image](https://cloud.githubusercontent.com/assets/3096584/16969844/f48591a6-4de5-11e6-9811-c251e643da04.png)
## Error Lag. (Using an E2 and 2 sound emitters)

We are simply using an E2 and 2 sound emitters to play 2 sounds that we don't have.
![image](https://cloud.githubusercontent.com/assets/3096584/16970184/4db54454-4de8-11e6-8d69-608179392f19.png)
## With _con_filter_enable 1; con_filter_text_out "Failed to load"_

![image](https://cloud.githubusercontent.com/assets/3096584/16970333/43436bc6-4de9-11e6-8c46-aaf53d7d0892.png)

Note: This is a clearer version of https://github.com/Facepunch/garrysmod-requests/issues/783 as it seems my initial post was misinterpreted.

Note: I'm not sure if this was better under suggestions, or issues. Because this is a suggestion on how to fix an issue.

**Comment by BoXiousNet** [comment]
https://github.com/Facepunch/garrysmod-issues/issues/869

**Comment by LuaTenshi** [comment]
Should I repost and refrence this in [Issues](https://github.com/Facepunch/garrysmod-issues/issues)? (As that's where this should be in the first place... oops...)

**Comment by willox** [comment]
Does this happen with anything other than sounds?

**Comment by BoXiousNet** [comment]
Also happens if a map overrides an existing model/material which doesn't exist on level change.

**Comment by LuaTenshi** [comment]
I'm not sure where I can get the above error for testing. But I will try to get some screenshots with +showbudget graph so we have something to go along with the net_graphs above.

**Comment by LuaTenshi** [comment]
@willox sorry for being super late on this but I don't check Garrysmod issues often. I've only noticed this happening with sounds if any one else can report that would be nice.

**Comment by robotboy655** [comment]
No idea why this is in requests, but this should be much better in Dev.
